packages/icons/package.json

```diff
@@ -1,36 +1,46 @@
 {
   "name": "@obosbbl/grunnmuren-icons",
   "version": "0.1.1",
-  "description": "Grunnmuren icons as SVGs",
+  "description": "Grunnmuren's icon set",
   "license": "MIT",
   "type": "module",
   "repository": {
     "url": "https://github.com/code-obos/grunnmuren",
     "directory": "packages/icons"
   },
   "exports": {
-    ".": {
-      "require": "./index.cjs",
-      "import": "./index.mjs"
-    },
+    "./react": "./dist/icons.es.js",
```

Comment:
Should the React icons be the default export, or should it use a subpath export? Most people will use the React components, and the package has a peer dependency on React...

So with that context, what makes more sense?
```jsx
import { House } from '@obosbbl/grunnmuren-icons/react';
// or
import { House } from '@obosbbl/grunnmuren-icons';
```

Reply:
Should others have the option of getting the svg files if they don't use react in their apps? I'm fine in giving react components the default export, but at least give others the option of using the svg files as well (like our old stack, if they want to implement it in vue)

Reply:
The SVGs are still exported from `@obosbbl/grunnmuren-icons/svg`